packages/ui-kit/src/design-system.ts

```diff
@@ -258,6 +264,7 @@ export const typography = {
   fontFamilies: {
     primary: "'Roboto', sans-serif",
     code: "'Roboto Mono', monospace",
+    documentationLogo: 'TTCommonsPro-Rg, TT Commons Pro',
```

Comment:
What's the plan on using this new fonts? Should they replace `Roboto`?

Also, you need to actually import them otherwise they won't be applied. Example https://github.com/commercetools/commercetools-docs-kit/blob/a4cd66c10a346aacb03cc1e019d2aba009168c14/packages/gatsby-theme-docs/gatsby-browser.js#L8-L13

Reply:
Unless I'm missing some context here, I would remove this change for now and keep it as it was until there is a formal decision.

Reply:
Yes we decided to keep the documentation logo as it is and to implement that separately.